src/Appwrite/Utopia/Response/Model/AttributeDatetime.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+
+class AttributeDatetime extends Attribute
+{
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this
+            ->addRule('key', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Attribute Key.',
+                'default' => '',
+                'example' => 'birthDay',
+            ])
+            ->addRule('type', [
+                'type' => self::TYPE_DATETIME,
+                'description' => 'Attribute type.',
+                'default' => '',
+                'example' => self::TYPE_DATETIME_EXAMPLE,
+            ])
+            ->addRule('format', [
+                'type' => self::TYPE_DATETIME,
+                'description' => 'Datetime format.',
+                'default' => APP_DATABASE_ATTRIBUTE_DATETIME,
+                'example' => APP_DATABASE_ATTRIBUTE_DATETIME,
+                'array' => false,
+                'require' => true,
+            ])
+            ->addRule('default', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Default value for attribute when not provided. Only null is optional',
```

Comment:
I don't think I'm understanding this description.  Do you mean that you should set this value to null when you don't want a default value?

Does this behave differently from other attributes? Other attributes's default parameter have the description: 
> Default value for attribute when not provided. Cannot be set when attribute is required.

Reply:
Yes this works different Datetime can not have every value as default.